man/pbdb_collection.Rd

```diff
@@ -31,8 +31,6 @@ the main filtering parameters.
 }
 \examples{
 \dontrun{
-pbdb_collection (id=1003, vocab="pbdb", show="loc")
-
+  pbdb_collection(id = 1003, vocab = "pbdb", show = "loc")
```

Comment:
Should be consistent with **R/pbdb_querys.R** **line 688**: pbdb_reference(id = 1003, vocab = "pbdb", show = "both")